securedrop/client.py

```diff
@@ -3,10 +3,14 @@
 import os
 import hashlib
 import base64
+import Crypto
+
+from Crypto import Random
```

Comment:
Since we only use `get_random_bytes`, we can just do `from Crypto.Random import get_random_bytes`.